test/webdriverio/test/insert_test.ts

```diff
@@ -15,7 +15,7 @@ import {
   testSetup,
 } from './test_setup.js';
 
-suite.only('Insert test', function () {
```

Comment:
Oops!

Would need a separate run step for CI vs local to sanity check this but might be worth it.

https://github.com/mochajs/mocha/issues/5299